src/client/components/Forms/SignUp.js

```diff
@@ -1,57 +1,171 @@
-import React, { useState } from 'react';
-import PropTypes from 'prop-types';
+import React from 'react';
+import useForm from '../LoginForm/useForm';
+import iconGithub from '../../assets/images/icons/github.svg';
+import './Signup.style.css';
+import Button from '../Button/Button';
+import validate from '../LoginForm/Validation';
 
 export default function SignUp({ onSubmit }) {
-  const [email, setEmail] = useState('');
-  const [password, setPassword] = useState('');
-  const [passwordConfirm, setPasswordConfirm] = useState('');
-  const handleEmailInput = (e) => setEmail(e.target.value);
-  const handlePasswordInput = (e) => setPassword(e.target.value);
-  const handlePasswordConfirmInput = (e) => setPasswordConfirm(e.target.value);
-  const handleSubmit = (e) => {
-    e.preventDefault();
-    onSubmit({ email, password, passwordConfirm, fullName: 'Jane Doe' });
-  };
+  const { handleChange, handleSubmit, values, errors } = useForm(
+    validate,
+    onSubmit,
+    {
+      fullName: '',
+      position: '',
+      profileImageUrl: '',
+      linkedin: '',
+      github: '',
+      website: '',
+      email: '',
+      password: '',
+      passwordConfirm: '',
+    },
+  );
+
   return (
-    <form onSubmit={handleSubmit}>
-      <label htmlFor="email">
-        Email:{' '}
+    <form className="form" onSubmit={handleSubmit} noValidate>
```

Comment:
```suggestion
    <form className="form" onSubmit={handleSubmit}>
```

Reply:
> 

its because from takes auto validation so our validation component doesnot exists therfore we need this.

Reply:
I believe you don't need it, because if we have it, if the user doesn't enter fill a field and clicks _Create account_, then a kind of alert appears and the sign-up page remains in loading. While if you don't have the `noValidate`, it will indicate the user in the same page, that a field has not being filled.
I did the exact same thing in both scenarios of these pictures: 

![Kamal2](https://user-images.githubusercontent.com/36211640/107634251-898cbd00-6c69-11eb-9db4-037bd5f43df2.jpg)
![kamal1](https://user-images.githubusercontent.com/36211640/107634254-8abdea00-6c69-11eb-9876-cfa9fc64fa3d.jpg)

Reply:
agreed!! and updated